const_windows.go

```diff
@@ -0,0 +1,8 @@
+// +build windows
```

Comment:
Fine with leaving it in for explicit-ness but for my own understanding, we don't technically need it when using the `*_windows.go` filename right?